lib/authentic/validator.rb

```diff
@@ -8,52 +8,33 @@ module Authentic
   # Public: validate JWTs against JWKs using iss whitelist in an environment variable.
   #
   # token - raw JWT.
-  # opts  - Optionally pass configuration options.
   #
   # Returns boolean.
-  def self.valid?(token, opts = {})
-    Validator.configure(opts) unless opts.empty?
+  def self.valid?(token)
```

Comment:
These two class methods (`self.valid?`, `self.ensure_valid`) should probably go away so we don't encourage people to use this in a manner that blows out the cache on every usage.